modules/components/pages/inputs/schema_registry.adoc

```diff
@@ -0,0 +1,285 @@
+= schema_registry
+:type: input
+:status: beta
```

Comment:
is this an official beta? Do we define beta for RP Connect anywhere?

Maybe worth adding an admonition to notify users of this status.

Reply:
There are many connectors with a BETA status, adding an admonition to just one might cause confusion?

Reply:
updated